### PR TITLE
Add IBMMQ exporter

### PR DESCRIPTION
**Pull request by gsanchezgavier**
Adds the ibmmq prometheus integration with e2e test

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,6 +41,6 @@ nri-config-generator/templates/*
 !nri-config-generator/definitions/index.md
 !nri-config-generator/templates/config.json.tmpl
 .env
-dist
+exporters/ibmmq/e2e/dist
```
Comment (gsanchezgavier) [inline .gitignore:44]: just saw this , it need to be gone

File: exporters/ibmmq/e2e/ibmmq.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    metrics: 
+      - name: ibmmq_qmgr_uptime
+        type: gauge
+        defaultResolution: 15
+        unit: count
+        dimensions:
+          - name: platform
+            type: string
+          - name: qmgr
+            type: string
```
Comment (alvarocabanas) [inline exporters/ibmmq/e2e/ibmmq.yml:17]: newline

File: exporters/ibmmq/e2e/producer-entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+i=0
+while true; do
+    echo Message $i | ./opt/mqm/samp/bin/amqsputc DEV.QUEUE.1
+    ((i++))
+    sleep .1
+done
```
Comment (alvarocabanas) [inline exporters/ibmmq/e2e/producer-entrypoint.sh:8]: newline

**Review by alvarocabanas — APPROVED** [review]
LGTM

**Pull request merged**